core/test/utils/index.js

```diff
@@ -251,12 +251,20 @@ fixtures = {
     // Creates a client, and access and refresh tokens for user 3 (author)
     createTokensForUser: function createTokensForUser() {
         return db.knex('clients').insert(DataGenerator.forKnex.clients).then(function () {
-            return db.knex('accesstokens').insert(DataGenerator.forKnex.createToken({user_id: 3}));
+            return db.knex('accesstokens').insert(DataGenerator.forKnex.createToken({id:3, user_id: 3}));
```

Comment:
wouldn't it make more sense to have the `createTokensForUser` ids be 1 and 2, and then the `createMoreTokens` ids be 3 and 4?